app/helpers/ukims_helper.rb

```diff
@@ -0,0 +1,25 @@
+module UkimsHelper
```

Comment:
Should we put a comment on what IMS stands for? (I did know what it was before this PR :) )

`# Helper for Internal Market Scheme`

Reply:
Yes this might be a good idea at the top im happy to add a comment 👍 , I dont mind class level comments if not to long.